Modix.Services/Core/MessageLogBehavior.cs

```diff
@@ -3,23 +3,30 @@
 using System.Threading.Tasks;
 using Discord;
 using Discord.WebSocket;
+using Microsoft.Extensions.Logging;
 using Modix.Data.Models.Core;
+using Modix.Data.Repositories;
 using Modix.Services.Utilities;
-using Serilog;
 
 namespace Modix.Services.Core
 {
     public class MessageLogBehavior : BehaviorBase
```

Comment:
Just so's we're clear, this will need to be re-written a bit when we formally adopt @NullEntity 's MediatR stuff.